src/axolotl/utils/config.py

```diff
@@ -358,9 +358,9 @@ def validate_config(cfg):
             "push_to_hub_model_id is deprecated. Please use hub_model_id instead."
         )
 
-    if cfg.hub_model_id and not (cfg.save_steps or cfg.saves_per_epoch):
+    if cfg.hub_model_id and cfg.save_strategy not in ["steps", "epoch", None]:
```

Comment:
I think this check was made to ensure that one of save_steps/saves_per_epoch/save_strategy was set. In this case, you're missing those conditions?

Reply:
Isen't the only thing that matters that you have set a save strategy? https://huggingface.co/docs/transformers/main_classes/trainer#transformers.TrainingArguments.save_strategy. And to not save you have to set it to NO as I have understood it, if you don't set it (None in axolotl) it becomes steps.

Reply:
Should this be changed to check for "no" then?

Reply:
It could be, but currently the test checks if it is no or any other value. I wanted to check more widely, but can tighten it up if you think it is better @NanoCode012 . Also sorry for slow progress :(